dotcom-rendering/scripts/webpack/webpack.config.browser.js

```diff
@@ -31,9 +32,15 @@ module.exports = ({ isLegacyJS, sessionId }) => ({
 			'./src/web/browser/newsletterEmbedIframe/init.ts',
 		relativeTime: './src/web/browser/relativeTime/init.ts',
 		initDiscussion: './src/web/browser/initDiscussion/init.ts',
+		debug: './src/web/browser/debug/init.ts',
 	},
 	output: {
-		filename: generateName(isLegacyJS),
+		filename: (data) => {
+			// We don't want to hash the debug script so it can be used in bookmarklets
+			if (data.chunk.name === 'debug')
+				return generateName(isLegacyJS, false);
```

Comment:
Could we remove the abstraction and just use:
```suggestion
				return `[name].js`
```

As we don't want to use this anywhere else and don't need to support chunkname nor legacy?

Reply:
Agree with this suggestion!

Reply:
Wouldn't doing it this way mean the legacy bundle would just overwrite the original bundle when generated, since we're giving it the same name?

Reply:
Does it matter? We don’t really need two files. But you could always turn it into a ternary operator.

An alternative would be to simply add a copy step and serve the files untransformed, without the Webpack step. You would have to import the css in a JS way, however, so this might be more effort.